source/configuration_manual/authentication/oauth2.rst

```diff
@@ -98,41 +113,33 @@ passdb settings
 
 .. code-block:: none
 
-  auth_oauth2_config_file = /usr/local/etc/dovecot/dovecot-oauth2.token.conf.ext
-
   passdb oauth2 {
     mechanisms = plain login
-    args = /usr/local/etc/dovecot/dovecot-oauth2.plain.conf.ext
-  }
 
-put into ``dovecot-oauth2.token.conf.ext``
-
-.. code-block:: none
-
-  grant_url = http://localhost:8000/token
-  client_id = verySecretClientId
-  client_secret = verySecretSecret
-  tokeninfo_url = http://localhost:8000/oauth2?oauth=
-  introspection_url = http://localhost:8000/introspect
-  introspection_mode = post
-  use_grant_password = no
-  debug = yes
-  username_attribute = username
-  pass_attrs = pass=%{oauth2:access_token}
-
-put into ``dovecot-oauth2.plain.conf.ext``
-
-.. code-block:: none
+    oauth2 {
+      # common settings can be inherited from global scope
```

Comment:
I guess it's trying to say that it's inheriting the oauth { .. } settings from the global scope. Would be clearer if it said "inherit common oauth2 settings from the global scope", also clearer if the global oauth { .. } settings were before this passdb